lisp/ess-custom.el

```diff
@@ -719,6 +719,38 @@ some.function(arg1,
 
 ")
 
+(defvar ess-arg-function-offset-continued nil
+  "Extra indent for function arguments when ( is folowed by arguments and
+a new line.
+
+If nil, the statements are indented at open-parenthesis following foo:
```

Comment:
Here you go. I used an opening backtick and a closing quote as I saw this was used elsewhere.